lib/Conversation/styles.scss

```diff
@@ -116,9 +127,24 @@ $conversation-meta-text-color: $color-brand-dark !default;
 }
 
 .form__actions {
+  position: relative;
   text-align: right;
 
   .form__actions {
     margin: 0 0 0 $layout-spacing-base/2;
   }
+}
+
+.form__drop-down {
+  position: absolute;
+}
+
+.form__button {
```

Comment:
That was my initial thought too, something like comment-form is super declarative. Two 🌮 s for that!